### PR TITLE
Syshcndr_kannada_patch

### PR DESCRIPTION
**Pull request by syshcndr**

### CODE REVIEW

**Comment by spielers** [comment]
Hey, @syshcndr Thanks for your contribution! 

best,
spielers

File: app/config/locale/translations/ka.countries.php [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+   'IE' => 'ಐರ್ಲೆಂಡ್',
+   'IR' => 'ಇರಾನ್',
+   'IQ' => 'ಇರಾಕ್',
+   'IS' => 'ಐಸ್ಲ್ಯಾಂಡ್',
+   'IL' => 'ಇಸ್ರೇಲ್',
+   'IT' => 'ಇಟಲಿ',
+   'JM' => 'ಜಮೈಕಾ',
+   'JO' => 'ಜೋರ್ಡಾನ್',
+   'JP' => 'ಜಪಾನ್',
+   'KZ' => 'ಕ Kazakh ಾಕಿಸ್ತಾನ್',
```
Comment (spielers) [inline app/config/locale/translations/ka.countries.php:91]: Translate this to Kannada properly

**Review by spielers — CHANGES_REQUESTED** [review]

File: app/config/locale/translations/ka.countries.php (hunk 1 above)
Comment (syshcndr) [reply app/config/locale/translations/ka.countries.php]: Thank you for the suggestion, I have fixed it now.

**Review by Kolkies — APPROVED** [review]

**Review by spielers — APPROVED** [review]
Thanks for your contributions @syshcndr.

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**